Pickers/AbstractActionSheetPicker.m

```diff
@@ -218,7 +254,7 @@ - (void)showActionSheetPicker {
 
 #pragma clang diagnostic push
 #pragma ide diagnostic ignored "UnavailableInDeploymentTarget"
-    if ([UIViewController instancesRespondToSelector:@selector(edgesForExtendedLayout)]) {
```

Comment:
@kashifhisam :+1: Thanks!  Sorry for delay, was in vacation. Everything seems good. But one question about this line. Why you delete this if statement?